sdk/aqueduct/integrations/mongodb_integration.py

```diff
@@ -57,6 +58,7 @@ def find(
                 Whether to run this operator lazily. See https://docs.aqueducthq.com/operators/lazy-vs.-eager-execution .
         """
         op_name = name or self._dag.get_unclaimed_op_name(prefix="%s query" % self.name())
```

Comment:
Can you update the docstring with the new argument? Can use this one to be consistent with mine:

```
            output:
                Name to assign the output artifact. If not set, the default naming scheme will be used.
```